cmd/vulcan-prowler/main.go

```diff
@@ -127,9 +136,12 @@ var (
 			"https://github.com/toniblyx/prowler",
 			"https://www.cisecurity.org/benchmark/amazon_web_services/",
 		},
-		Score: report.SeverityThresholdMedium,
+		Score: report.SeverityThresholdLow,
```

Comment:
Why the severity of the issue for level 2 is lower than for level 1?

Reply:
That's a good question, we agreed on considering not being compliant with the CIS Level 2 not as bad as not being compliant with the CIS Level 1, which is more basic. It is true that there is no clear solution here.
What's your opinion regarding this?